dart/dynamics/FreeJoint.cpp

```diff
@@ -508,25 +508,9 @@ void FreeJoint::setAngularAcceleration(
 
 //==============================================================================
 Eigen::Matrix6d FreeJoint::getLocalJacobianStatic(
-    const Eigen::Vector6d& _positions) const
+    const Eigen::Vector6d& /*positions*/) const
```

Comment:
Yes. Previously a Jacobian of FreeJoint was dependent on joint positions like other joints but it's only dependent on the transform from child BodyNode to the joint.